runtime/tr.source/trj9/build/scripts/j9jit.rc

```diff
@@ -40,12 +40,12 @@ BEGIN
 		BEGIN
 			VALUE "CompanyName", "International Business Machines Corporation\0"
 			VALUE "FileDescription", "J9 Virtual Machine Runtime\0"
-			VALUE "FileVersion", "R2.8 (" EsBuildVersionString ")\0"
-			VALUE "InternalName", "j9jit28\0"
-			VALUE "LegalCopyright", " � Copyright (c) 1991, 2014 IBM Corp. and others\0"
-			VALUE "OriginalFilename", "j9jit28.dll\0"
+			VALUE "FileVersion", "R" EsVersionString "(" EsBuildVersionString ")\0"
```

Comment:
Why introduce `EsVersionString` here when the `2.9` are hard coded everywhere else in this file?

Reply:
I considered replacing this file with j9jit.rc.ftl to gain direct access to all UMA properties, but decided against that because we're trying to stop using UMA.

I used a macro where I could find an existing definition. The expansion of `EsVersionString` is `"2.9"`; I couldn't find one that expanded to `"29"`. Would you prefer I leave those things hard-coded? (I don't have a strong preference either way.)

Reply:
I would leave it hard coded so the values will be consistent within the file.

Reply:
Use of `EsVersionString` removed.